cmlreaders/cmlreader.py

```diff
@@ -0,0 +1,51 @@
+from .readers import *
+
+
+__all__ = ['CMLReader']
+
+
+class CMLReader(BaseCMLReader):
+    readers = {
+            'voxel_coordinates': TextReader
+        }
+
+    def __init__(self, subject=None, experiment=None, session=None,
+                 localization=None, montage=None, rootdir="/"):
+        """ Instatiates a general reader for CML-specific data """
+
+        self.subject = subject
+        self.experiment = experiment
+        self.session = session
+        self.localization = localization
+        self.montage = montage
+        self.rootdir = rootdir
+
+        # File type and reader are unintialized initially, but are populated
+        # when the user requests that a particular file be loaded
+        self.file_type = None
+        self._reader = None
+
+    def load(self, file_type):
+        self.file_type = file_type
+        self._reader = self.readers[self.file_type](self.file_type,
```

Comment:
I'm thinking it makes more sense to check if the requested data type is a key in the readers dict and raise a NotImplemented error instead of a generic key error. Let me know if you disagree.

Reply:
That sounds fine.